### PR TITLE
WebXRManager: immersivestart/end events

### PR DESCRIPTION
**Pull request by takahirox**
From #17737, please refer to it for the detail.

This PR adds two changes
- Rename sessionstart/end events to immersivestart/end events
- Fire immersivestart event in the first animation frame from WebXRManager

### CODE REVIEW

File: src/renderers/webvr/WebVRManager.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ function WebVRManager( renderer ) {
 
 			animation.start();
 
-			scope.dispatchEvent( { type: 'sessionstart' } );
+			scope.dispatchEvent( { type: 'immersivestart' } );
```
Comment (Mugen87) [inline src/renderers/webvr/WebVRManager.js]: `WebVRManager` has been removed. Can you please rebase?

**Comment by Mugen87** [comment]
If this change is still wanted, please resolve the merge conflicts. Otherwise it's better to close it and #17737.

**Comment by takahirox** [comment]
Let me review myself because I made this PR long time ago, and if it looks still necessary I would resolve the conflicts soon.

**Comment by Mugen87** [comment]
Feel free to reopen the PR if it is still necessary and the merge conflicts are solved.

Since a separate issue is not required, closing  #17737.

**Pull request closed without merging**